renderer-macos/lib/stubs/BriskStylableText.c

```diff
@@ -0,0 +1,107 @@
+#import "BriskStylableText.h"
+
+void ml_BriskStylableText_applyChanges(id<BriskStylableText> txt) {
+  [txt applyTextStyle];
+}
+
+CAMLprim value ml_BriskStylableText_setFont(id<BriskStylableText> txt,
+                                            value fontName_v, double fontSize,
+                                            double fontWeight) {
+  CAMLparam1(fontName_v);
+
+  NSString *fontName = [NSString stringWithUTF8String:String_val(fontName_v)];
+  NSFont *font;
+
+  if (fontName.length > 0) {
+    NSDictionary *attributes = @{
+      NSFontFaceAttribute : fontName,
+      NSFontTraitsAttribute :
+          @{NSFontWeightTrait : [NSNumber numberWithDouble:fontWeight]}
+    };
+
+    NSFontDescriptor *descriptor =
+        [NSFontDescriptor fontDescriptorWithFontAttributes:attributes];
+
+    font = [NSFont fontWithDescriptor:descriptor size:(CGFloat)fontSize];
+  } else {
+    font = [NSFont systemFontOfSize:(CGFloat)fontSize weight:fontWeight];
+  }
+
+  txt.attributedProps[NSFontAttributeName] = font;
+
+  CAMLreturn(Val_unit);
+}
+
+void ml_BriskStylableText_setColor(id<BriskStylableText> txt, double red,
+                                   double green, double blue, double alpha) {
+  txt.attributedProps[NSForegroundColorAttributeName] =
+      [NSColor colorWithRed:red green:green blue:blue alpha:alpha];
+}
+
+CAMLprim value ml_BriskStylableText_setColor_bc(id<BriskStylableText> txt,
+                                                value red_v, value green_v,
+                                                value blue_v, value alpha_v) {
+  CAMLparam4(red_v, blue_v, green_v, alpha_v);
+
+  ml_BriskStylableText_setColor(txt, Double_val(red_v), Double_val(blue_v),
+                                Double_val(green_v), Double_val(alpha_v));
+
+  CAMLreturn(Val_unit);
+}
+
+void ml_BriskStylableText_setAlignment(id<BriskStylableText> txt, int align) {
+  txt.paragraphStyle.alignment = align;
```

Comment:
Elaborate? It's a mutable property on our custom instance. We re-apply `paragraphStyle` when we commit style changes.

Reply:
Same as before - we pass arbitrary ints from OCaml. They might not match the constants defined in Objc.

Reply:
Aah, you were referring to the `int`!
This is the easiest/fastest way, we could create an issue and review this at some point.

Reply:
I think this sets wrong precedence. No need to rush PRs. We can take shortcuts but we should also set solid foundation.

Reply:
My thinking was that all platforms will use polymorphic variants for props and implement what they actually set.

If the values change and this starts causing issues, it's arguably easier to adjust them in OCaml bindings than deal with the mess in Obj-C.

Reply:
I mean this is inherently unsafe. We should use the enums directly, not their int values. This is begging for mistakes.

Reply:
I don't disagree, but from my perspective this looked safer than dealing with polymorphic variants on Obj-C side - we either deal with multiple if-then-else-if-then-else on hashes or bring in another variant just so we could map it on our own enums that will map to Cocoa's constants (not always enums!), yet still need to account for some quirks, because Apple decided to flip constant values for iOS.

Reply:
Not necessarily. One solution would be doing:
```
 `Ultralight => BriskText.ultralight()

type fontWeight;
[@noalloc] external ultralight: unit => fontWeight = "ml_BriskText_ultralight";
```